src/audio/host.c

```diff
@@ -746,10 +746,10 @@ static int host_pointer_reset(struct comp_dev *dev)
 static int host_stop(struct comp_dev *dev)
 {
 	/* reset host side buffer pointers */
-	host_pointer_reset(dev);
+	//host_pointer_reset(dev);
 
 	/* reset elements, to let next start from original one */
-	host_elements_reset(dev);
+	//host_elements_reset(dev);
```

Comment:
I think pointers reset by this function aren't used for Host DMA anyway, so I doubt it changes anything on APL.